.github/workflows/cleanup-snapshots.yml

```diff
@@ -0,0 +1,45 @@
+name: Cleanup Snapshot Artifacts on a CRON Schedule
```

Comment:
We should document this in our [Automation Guide.](https://docs.cplace.io/frontend-applications/guides/automation/)